index.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
```

Comment:
divide the body using section or div in order to differentiate between contents.

Reply:
Hello, @betses ✋

- [ ] You may please make this comment more of a suggestion than an order.